packages/e2e/cypress/support/commands.d.ts

```diff
@@ -3,30 +3,39 @@
 // load the global Cypress types
 /// <reference types="cypress" />
 
-declare namespace Cypress {
-  interface Chainable<Subject> {
-    /**
-     * Custom command to select input element by role or placeholder text
-     *
-     * @param {string} field - name of the input field, or alternatively
-     *   placeholder value for password inputs.
-     * @example cy.findInputField('Password')
-     */
-    findInputField(field: string): Chainable<JQuery<HTMLInputElement>>;
-  }
-}
+declare global {
```

Comment:
This refactor is to follow the [recommended approach in the Cypress docs](https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfig-json).